roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -74,6 +74,9 @@ kubernetesMasterConfig:
   masterCount: 1
   masterIP: ""
   podEvictionTimeout: ""
+  proxyClientInfo:
```

Comment:
Thus far we've not conditionalized bits in the template if we knew they'd be ignored under older versions. I can see how that'd be confusing to users but we mostly assume that if they're running the latest ansible code they're probably going to be running the latest product code.